### ISSUE TITLE
Q-value

### ISSUE THREAD

**Issue by boyeon-kim** [yoon-gu/ezlab-rl#33]
For initial weight, Q-value 계산

**Comment by boyeon-kim** [comment]
Step1. target action에 대한 Q-value 계산
Step2. 동일한 Q-network 구축
Step3. loss = mse(Q-value - Target Q-value)
Step4. minimizer 찾기..?!
